<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiagnosticsPage</class>
 <widget class="QWidget" name="DiagnosticsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Run</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>File/Run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="run_input_line_edit"/>
      </item>
      <item>
       <widget class="QPushButton" name="browse_button">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Period:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="period_line_edit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>/n</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Detector1</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="3">
       <widget class="QCheckBox" name="det1_horizontal_mask_check_box">
        <property name="text">
         <string>Apply Time Channel Mask</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="detector_1_vertical_integral_button">
        <property name="text">
         <string>Vertical Integral</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Range:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Range:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="det1_vertical_range_line_edit"/>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="detector_1_horizontal_button">
        <property name="text">
         <string>Horizontal Integral</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="det1_vertical_mask_check_box">
        <property name="text">
         <string>Apply Time Channel Mask</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="det1_horizontal_range_line_edit"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="detector_1_time_integral_button">
        <property name="text">
         <string>Time Integral</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Range:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="det1_time_range_line_edit"/>
      </item>
      <item row="2" column="3">
       <widget class="QCheckBox" name="det_1_time_mask_checkbox">
        <property name="text">
         <string>Apply Pixel Mask</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Detector2</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="3">
       <widget class="QCheckBox" name="det_2_time_mask_checkbox">
        <property name="text">
         <string>Apply Pixel Mask</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="det2_vertical_mask_check_box">
        <property name="text">
         <string>Apply Time Channel Mask</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Range:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Range:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="detector_2_time_integral_button">
        <property name="text">
         <string>Time Integral</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Range:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="det2_horizontal_mask_check_box">
        <property name="text">
         <string>Apply Time Channel Mask</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="det2_vertical_range_line_edit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="det2_horizontal_range_line_edit"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="det2_time_range_line_edit"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="detector_2_vertical_integral_button">
        <property name="text">
         <string>Vertical Integral</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="detector_2_horizontal_button">
        <property name="text">
         <string>Horizontal Integral</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
